oauth2_provider_jwt/utils.py

```diff
@@ -60,11 +63,16 @@ def decode_jwt(jwt_value):
     payload_enc += '=' * (-len(payload_enc) % 4)  # add padding
     payload = json.loads(base64.b64decode(payload_enc).decode("utf-8"))
 
-    public_key_name = 'JWT_PUBLIC_KEY_RSA_{}'.format(payload['iss'].upper())
-    public_key = getattr(settings, public_key_name, None)
-    if not public_key:
+    algorithms = getattr(settings, 'JWT_JWS_ALGORITHMS', ['HS256', 'RS256'])
+    secret_name = 'JWT_PUBLIC_KEY_{}'.format(payload['iss'].upper())
```

Comment:
Why do you rename the `public_` and `private_key` to secret? It’s not so explicit anymore.

Reply:
I only renamed the private-key to `secret`, to allow the same field to be configured even if you are using the HMAC-suite(symetric).
If it where the old naming scheme I think it would be confusing for these users to put their secret in both fields.
On my side it's not a real problem to use the old scheme, but then you would have to do with every function-call something like this

```pyhton
if algo.startswith('HS'):
    key = getsetting('JWT_SECRET_')
else:
    key = getsetting('JWT_PUBLIC_KEY_')
```

I think this is unnecessary and on high payload services only slows everything down.

Reply:
Sorry realized just now, that you mean the internal variables -.-

Well basically for the same reason but renaming them again isn't that much of a problem^^